core/sdk/src/framework/ripple_contract.rs

```diff
@@ -175,6 +180,7 @@ impl RippleContract {
         match self {
             Self::Storage(adj) => Some(adj.as_string()),
             Self::Session(adj) => Some(adj.as_string()),
+            Self::MockServer(adj) => Some(adj.as_string()),
```

Comment:
Adding all of these adjective entries caught me out at first. I wonder if there is a way to get the compiler to tell use that we have not updated all the required functions.